test/jdk/tools/jlink/runtimeImage/ModifiedFilesWithShaOverrideTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2025, Red Hat, Inc.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.nio.file.Path;
+
+/*
+ * @test
+ * @summary Verify no warnings are being produced on a modified file which
+ *          gets the SHA override from command line
+ * @requires (vm.compMode != "Xcomp" & os.maxMemory >= 2g & os.family == "linux")
+ * @library ../../lib /test/lib
+ * @enablePreview
```

Comment:
Do you know why @enablePreview is necessary here?

Reply:
It escapes me now and a bit of digging I think this was due to the fact JEP 493 being developed along side JEP 484 (ClassFile API). At the time some helpers in jlink tests used the ClassFile API and was still in preview at the time.

It no longer seems to be needed, so let me remove those.